migrator/runner/all.go

```diff
@@ -54,4 +54,60 @@ import (
 	_ "github.com/stackrox/rox/migrator/migrations/m_97_to_98_exclude_oauth_sa_kubeadmin_pol"
 	_ "github.com/stackrox/rox/migrator/migrations/m_98_to_m_99_process_alert_comments"
 	_ "github.com/stackrox/rox/migrator/migrations/m_99_to_m_100_violation_report_branding"
+	_ "github.com/stackrox/rox/migrator/migrations/n_01_to_n_02_postgres_clusters"
```

Comment:
Recommend adding this to a different package called all_postgres.go to just separate them